src/Resources/skeleton/message/Message.tpl.php

```diff
@@ -0,0 +1,32 @@
+<?= "<?php\n" ?>
+
+namespace <?= $namespace; ?>;
+
+class <?= $class_name."\n" ?>
+{
+    /*
+     * Add whatever properties & methods you need to hold the
```

Comment:
what about doing it like in the make entity command, one could nearly reuse the existing code without the annotations stuff. This would make it more handy imo

Reply:
Hi, can you point what part of command or entity can be reused? :)